File: Results/DER/Translation/translation_output/w_test/gpt-4/codenet/python/java/p00388.java
Here is the equivalent Java code for the provided Python script. The Java code includes taking inputs from the user, processing the information, and then outputting the result.

```java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int h = scanner.nextInt();
        int a = scanner.nextInt();
        int b = scanner.nextInt();
        scanner.close();

        int count = 0;
        for (int c = 0; c <= b - a; c++) {
            if (h % (a + c) == 0) {
                count++;
            }
        }

        System.out.println(count);
    }
}
```

You can use this Java code by running it inside a Java environment. It will perform the same functionality as the Python code provided, which calculates the number of occurrences where `(h % (a + c))` equals `0` for `c` in the range from `0` to `b - a` inclusive.

File: Results/DER/Translation/translation_output/w_test/gpt-4/codenet/java/python/p00388.py
Here is the Python translation of the provided Java code:

```python
# Design of a Mansion in Python

def main():
    H, A, B = map(int, input().split())
    ans = 0

    for i in range(A, B + 1):
        if H % i == 0:
            ans += 1

    print(ans)

if __name__ == "__main__":
    main()
```

You can test it with the provided test input:

Test input:
```
100 2 4
```

Expected output:
```
2
```
